libuavcan_drivers/stm32/driver/include/uavcan_stm32/thread.hpp

```diff
@@ -17,6 +17,9 @@
 # include <ctime>
 # include <cstring>
 #elif UAVCAN_STM32_BAREMETAL
+#elif UAVCAN_STM32_FREERTOS
+# include <stm32f4xx.h>
```

Comment:
Can we replace this with a more generic include that doesn't depend on a particular MCU model?